package.json

```diff
@@ -131,27 +138,32 @@
       ],
       "explorer/context": [
         {
-          "when": "resourceFilename =~ /[dD]ocker[fF]ile/",
+          "when": "resourceFilename =~ /(^|\\.)dockerfile$/i",
           "command": "vscode-docker.acr.quickBuild",
           "group": "docker"
         },
         {
-          "when": "resourceFilename =~ /[dD]ocker-[cC]ompose/",
+          "when": "resourceFilename =~ /^(?:(?!^docker-compose\\.ya?ml$).)*\\.ya?ml$/i",
+          "command": "vscode-docker.acr.runTaskFile",
+          "group": "docker"
+        },
+        {
+          "when": "resourceFilename =~ /^docker-compose\\.ya?ml$/i",
           "command": "vscode-docker.compose.down",
           "group": "docker"
         },
         {
-          "when": "resourceFilename =~ /[dD]ocker-[cC]ompose/",
+          "when": "resourceFilename =~ /^docker-compose\\.ya?ml$/i",
```

Comment:
The case-insensitivity is great, but please don't change the actual patterns right now. There are already issues around this, and users can be very picky about what gets picked up.

Reply:
I changed this to match the current vs-code Iconography. This way docker-compose files will match everything vs-code currently detects as a docker-compose file in the sidebar.

Should I still change it?

Reply:
Yeah, I think it best not to mess with it yet. Your pattern is more restrictive than the original. We need to look at all these issues together.